Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.py

```diff
@@ -0,0 +1,107 @@
+from CommonServerPython import *  # noqa: F401
+
+
+import ipaddress
+import socket
+
+
+def snmp_send(udp_socket, probe, host, port):
+    """Return the Snmp response
+
+        Parameters:
+            udp_socket (object): Python object of socket.
+            probe (str): String value of Snmp data payload.
+            host (str): String value of IP address.
+            port (int): Integer port number.
+
+        Returns:
+            Str: Returns the value of the Snmp response.
+    """
+
+    udp_socket.sendto(bytes.fromhex(probe), (host, port))
+    udp_socket.settimeout(3)
```

Comment:
add `timeout` argument to the script ad use it here

Reply:
Passing the time out as an argument. Reference code is [here](https://github.com/demisto/content/blob/4a6e04b4e0fc5d3a1c73748435bde9fba4f2d281/Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.py#L97)